bedrock/mozorg/templates/mozorg/contribute/tasks/v1/build-firefox.html

```diff
@@ -1,28 +0,0 @@
-{# This Source Code Form is subject to the terms of the Mozilla Public
```

Comment:
Hmmmm, have we switched of the experiment or, can people still get to the alternate signup page? If so, these will most likely 404